template/fullstack/base-sessions/client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
We have `yarn` here instead of `npm`. I believe this had been fixed in a previous PR :v: